lib/xform.js

```diff
@@ -41,18 +77,251 @@ var XFormState = State.extend(function (self, name, opts) {
 
     // Defaults
     opts = _.defaults(opts || {}, {
-        next:null,
+        next: null,
+        xforms_service_url: 'prd-vumi-xforms.aws.prk-host.net',
```

Comment:
We need to replace the xforms service with a special resource of some kind that allows access to specific internal servers, rather than generic HTTP access to everything.

@justinvdm and I had an idea for a "single-server-and-port" HTTP resource type that could be used generically for such requests.